src/browser/modules/Sidebar/project-files.constants.ts

```diff
@@ -33,6 +33,36 @@ export enum EXECUTE_COMMAND_ORIGINS {
   NONE = ''
 }
 
+export interface ProhibitedFilenameErrors {
+  chars: string[]
+  tests: RegExp[]
+}
+
+interface ProhibitedFilenamePlatform {
+  platform?: string
+}
+
+interface ProhibitedFilenameChar extends ProhibitedFilenamePlatform {
+  char: string
+  test?: never
+}
+
+interface ProhibitedFilenameTest extends ProhibitedFilenamePlatform {
+  test: RegExp
+  char?: never
+}
+
+export const PROHIBITED_FILENAME_CHAR_TESTS: (
+  | ProhibitedFilenameChar
+  | ProhibitedFilenameTest
+)[] = [
+  { char: '/' },
+  { char: '\\' },
+  { char: '..' },
+  { test: /^[.|?]/ },
+  { char: ':', platform: 'Win32' }
```

Comment:
does it not make send to remove them for all platforms just to avoid confusion?

Reply:
yeah probably